x-pack/legacy/plugins/siem/public/components/timeline/body/events/stateful_event.tsx

```diff
@@ -98,69 +110,91 @@ export class StatefulEvent extends React.Component<Props, State> {
       timelineId,
       toggleColumn,
       updateNote,
-      width,
     } = this.props;
 
     // If we are not ready to render yet, just return null
     // see componentDidMount() for when it schedules the first
     // time this stateful component should be rendered.
     if (!this.state.initialRender) {
-      return null;
+      return <EmptyRow style={{ height: '27px' }}></EmptyRow>;
     }
+
     return (
-      <TimelineDetailsComponentQuery
-        sourceId="default"
-        indexName={event._index!}
-        eventId={event._id}
-        executeQuery={!!this.state.expanded[event._id]}
+      <VisibilitySensor
+        partialVisibility={true}
+        scrollCheck={true}
+        offset={{ top: 50, bottom: -500 }}
```

Comment:
perhaps a `const` with a `jsdoc` comment explaining the purpose of the `-500` here would be useful?